unit-tests/live/frames/test-fps-permutations.py

```diff
@@ -0,0 +1,304 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+# test:donotrun:!nightly
+# test:timeout 750
```

Comment:
IMO 12.5 min is too much.
Let's drop IMU and keep this test w/o it.

Reply:
I think we should keep IMU, it can reveal problems we don't see with video only. Will be important to check on the D555e as IMU high frequency streaming can interrupt video.

Reply:
We have several options, but we do need to reduce the test time IMO.
* Take only default profiles of each sensor (we can miss some valid use cases check)
* We can stream all and not permutations (we will lose the ability to distinguish between failed scenarios)
* We can reduce test stream duration

What do you think?

Reply:
Is there any downside to reducing test stream time? I tried with 1 sec for steady state + 5 for streaming and the results still seem to be fine. This would take 3-4 minutes with 8 stream types.

Reply:
If you are testing with high FPS, say at least 30Hz, reducing the streaming time should be OK.
1 second for steady state might not be enough, there are models that time to first frame can be up to 2.5 seconds. reduce to 3 seconds.

I don't understand how you got 3 minutes. ( (8 choose 2) +1 ) * 6 [seconds] still gives more then 5.5 minutes.
Using 3 seconds steady state will give 7.6 minutes.

Reply:
Maybe we can test only one IR, then we will have ((7 choose 2) +1 * 8 seconds) = 344 seconds.

Reply:
After discussion. 1. We will only test one IR stream 2. I miscalculated the time it takes.
Reducing the collection time to 5 and steady state 3 seconds is OK.
Time to run ((7 choose 2) +1 * 8 seconds) = 176 seconds.